packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -11,6 +11,7 @@ import 'package:flutter/rendering.dart';
 
 import 'debug.dart';
 import 'focus_manager.dart';
+import 'widget_inspector.dart';
```

Comment:
It feels strange that the "framework.dart" depends on the inspector?

Reply:
potentially we should refactor the inspector library into a portion that makes sense for general debugging and a part that is specifically for the widget inspector. That is made more challenging as Dart currently lacks proper support for package_private so refactoring the inspector would require making the _Location class public which isn't desirable at least until it switches to use uris instead of absolute paths.

Reply:
This is currently not avoidable, do we want to wait for refactor?